src/CovidAppointmentTable.js

```diff
@@ -90,13 +141,24 @@ export default function CovidAppointmentTable() {
 				}
 				label="Only show locations with available appointments"
 			/>
+			<div >
+				<TextField id="outlined-basic" label="Enter Zipcode" variant="outlined" onChange={e => setZipcode(e.target.value)} value={zipcode}/>
+				<Button className={classes.filterButtons} variant="contained" color="secondary" onClick={getZipcodeLocation}>Search By Zip</Button>
+				<Button className={classes.filterButtons} variant="contained" color="secondary" startIcon={<GpsFixedIcon/>} onClick={getCurrentLocation}>Use My Location</Button>
+			</div>
+			{!loading && data.length === 0 ?
+				<div className={classes.notFoundBox}>
+					<h2 className={classes.locationsNotFound}>No Locations Found</h2>
+					<Button className={classes.filterButtons} variant="contained" color="secondary" onClick={resetData}>Show All Locations</Button>
```

Comment:
Since this will be a third type of filter (availability, vaccine type, and now ZIP), I've suggested an alternative, more unified UI. LMK what you think: #13